File: testing_codes/Python/3-SQL_Injection.py
import sqlite3

def main():
    conn = sqlite3.connect('test.db')
    c = conn.cursor()

    print("Opened database successfully")

    name = input("Enter name: ")
    password = input("Enter password: ")

    sql = "INSERT INTO Users (name, password) VALUES ('{}', '{}')".format(name, password)
    try:
        c.execute(sql)
        conn.commit()
        print("Records created successfully")
    except sqlite3.Error as e:
        print("SQL error:", e)
    finally:
        conn.close()

if __name__ == "__main__":
    main()

File: testing_codes/Java/3-SQL_Injection.java
import java.sql.*;

public class Main {
    public static void main(String[] args) {
        Connection conn = null;
        Statement stmt = null;
        try {
            Class.forName("org.sqlite.JDBC");
            conn = DriverManager.getConnection("jdbc:sqlite:test.db");
            System.out.println("Opened database successfully");

            String name = "";
            String password = "";

            System.out.print("Enter name: ");
            name = System.console().readLine();
            System.out.print("Enter password: ");
            password = System.console().readLine();

            stmt = conn.createStatement();
            String sql = "INSERT INTO Users (name, password) VALUES ('" + name + "', '" + password + "')";
            stmt.executeUpdate(sql);

            System.out.println("Records created successfully");
        } catch (Exception e) {
            System.err.println(e.getClass().getName() + ": " + e.getMessage());
            System.exit(0);
        } finally {
            try {
                if (stmt != null) stmt.close();
                if (conn != null) conn.close();
            } catch (SQLException e) {
                e.printStackTrace();
            }
        }
        System.out.println("Operation done successfully");
    }
}
